Keywords.txt

```diff
@@ -1,3 +1,4 @@
+Sweltered | Increases damage NPCs take by 35%, while also dealing 20 damage-over-time | 255 | 100 | 35
```

Comment:
More a general design thing, but it may be a good idea to generalize 'exposure' as its own keyword for effects like damage increases. It may also be worthwhile trying to unify the system in the backend so that sources can stack additively like players would expect instead of multiplicatively.